experiment/simulation/index.html

```diff
@@ -1,43 +1,185 @@
+                    </label>
+                </div>
+
+
+                <div class="v-datalist-title" id="top-border">Camera Options</div>
+                <p>
+
+                    <button id="add-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive is-info" onclick="addShape()">Add</button>
+                    <button id="edit-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive  is-success" onclick="editShape()">Edit</button>
+                    <button id="delete-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive  is-danger" onclick="deleteShape()">Delete</button>
+
+                    <div id="edit-modal" class="modal is-responsive">
+                        <div class="modal-content">
+                            <span class="close">&times;</span>
+                            <p>
+                                <p class="v-datalist-title"> Edit Shape Menu</p>
+                                <div id="Addshape" class="control" value="hi">
+                                    <div class="select" style="float:left;">
+                                        <select name="" id="shape-edit-dropdown" value="dropdown">
+                                    <option value="Tetrahedron">Tetrahedron</option>
+                                    <option value="Octahedron">Octahedron</option>
+                                    <option value="Dodecahedron">Dodecahedron</option>
+                                    <option value="Cube">Cube</option>
+                                </select>
+                                    </div>
+                                </div>
+                                </br>
+                                </br>
+
+                                <input class="input " style="width: 200px; float:left" id="x" type="number" placeholder="X" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="y" type="number" placeholder="Y" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="z" type="number" placeholder="Z" step=".01" value="0">
+
+                                <div class="control">
+                                    <button class="buttonisprimary" style="width: 200px; float:left; margin: 1rem;">Edit</button>
+                                </div>
+                            </p>
+                        </div>
+
+                    </div>
+
+                    <div id="add-modal" class="modal is-responsive">
+                        <div class="modal-content">
+                            <span class="close">&times;</span>
+                            <p>
+                                <p class="v-datalist-title"> Add Shape Menu</p>
+                                <div id="ShapeAdd" class="control">
+                                    <div class="select" style="float:left;">
```

Comment:
Please use flexbox in place of using float

Reply:
removed instances where float is being used